lib/isomorphic/wrappers/md.js

```diff
@@ -1,4 +1,6 @@
-import React, {PropTypes} from 'react'
+'use strict'
```

Comment:
You are most likely correct and being that I wasn't certain I decided to enforce it across the whole project. I'll reverse this change however given that it doesn't make much sense now.